packages/web3-eth/types/index.d.ts

```diff
@@ -83,17 +83,14 @@ export class Eth {
     ): Subscription<Log>;
```

Comment:
just murder the last overload on like 96 and should be all good to merge.

Reply:
Thanks @joshstevens19!